index.html

```diff
@@ -69,6 +70,43 @@ <h4>Enterprise</h4>
           </div>
         </div>
       </section>
+
+      <section class="clients">
+        <div class="clients__testimonials">
+          <h4>Testimonials</h4>
```

Comment:
should be `h3` , the tags are for semantics and we can't skip h3 by going from h2 to h4 directly, we can always change the styling with css

Reply:
changed, thanks for the explanation